spec/actor_spec.rb

```diff
@@ -1,5 +1,10 @@
 # frozen_string_literal: true
 
+# Is there a way to make that more easier to load?
+# I'm thinking about having explicit dependecies between example
+# files (require_relative) and some `Dir[].each { |f| require f }`
+# magic in the spec_helper.
```

Comment:
Agreed! Since some examples depend on others (`play SomeOtherActor, …`), I'd have to explicitely call `require` in these, though. (Or, perhaps better still, not have them depend on other examples.)